api/docker/docker

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -e
 
-python manage.py migrate
+python src/manage.py migrate
```

Comment:
Bit suspicious of why I need to do this?

Reply:
I'm ok with it staying in /src I don't see any need to move that?

Reply:
Other than aesthetics, no, not really. It's not important, just feel like we should do it as `src` to me, gives the impression of code in a development environment, not being run in a production container.

Reply:
What directory name would be betteR?

Reply:
It should just sit directly in the `app` directory imo.